src/shipchain_common/exceptions.py

```diff
@@ -212,3 +212,16 @@ def __init__(self, detail, status_code=None, code=None):
 
         if status_code:
             self.status_code = status_code
+
+
+class URLShortenerError(APIException):
```

Comment:
This is the third APIException class with an overridden __init__ with duplicate code. Could we make a generic exception class to DRY that up?